src/metadata/gas.rs

```diff
@@ -41,8 +41,8 @@ pub enum GasMetadataError {
     CostError(#[from] CostError),
     #[error("Not enough gas to run the operation. Required: {required_gas}, Available: {available_gas}.")]
     NotEnoughGas {
-        required_gas: u128,
-        available_gas: u128,
+        required_gas: Box<u128>,
+        available_gas: Box<u128>,
```

Comment:
I meant a single `Box` for both. That makes the variant's data take only one machine word.

Reply:
My bad just fixed